docs/pages/setup/reference/cli.mdx

```diff
@@ -688,6 +688,30 @@ Environment variables configure your tsh client and can help you avoid using fla
 | TELEPORT_ADD_KEYS_TO_AGENT | Specifies if the user certificate should be stored on the running SSH agent | yes, no, auto, only |
 | TELEPORT_USE_LOCAL_SSH_AGENT | Disable or enable local SSH agent integration | true, false |
 
+### tsh configuration file
+
+`tsh` has an optional configuration file that is stored in `$HOME/.tsh/config/config.yml`.
```

Comment:
Instead of `$HOME/.tsh` I would say `$TELEPORT_HOME/config/config.yaml` and mention that $TELEPORT_HOME defaults to $HOME/.tsh.

Also, is it ".yml" or ".yaml"?